src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -533,10 +533,10 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_zero_padded_conv_backprop_
         pad_input, pad_value, Shape{0, 0, 0, 0}, Shape{0, 0, 0, 0}, Shape{0, 0, 0, 0});
     auto pad_label = std::make_shared<pattern::op::Label>(pad, nullptr, NodeVector{pad});
 
-    auto output_delta = std::make_shared<pattern::op::Label>(element::f32, Shape{1, 1, 1, 1});
+    auto output_delta = std::make_shared<pattern::op::Label>(element::f32, Shape{1, 1, 3, 3});
```

Comment:
@aprocter Shape{1, 1, 1, 1} seems more accurate?

Reply:
I think you're right, and this test was getting caught up in the same error that @postrational identified. I'll give it a try and likely revert.

Reply:
Confirmed- this has been reverted